dwi_ml/data/hdf5/hdf5_creation.py

```diff
@@ -598,9 +628,11 @@ def _create_streamline_groups(self, ref, subj_input_dir, subj_id,
 
             if len(sft.data_per_point) > 0:
                 logging.debug('sft contained data_per_point. Data not kept.')
-            if len(sft.data_per_streamline) > 0:
-                logging.debug('sft contained data_per_streamlines. Data not '
-                              'kept.')
+
+            for dps_key in self.dps_keys:
+                logging.debug("    Include dps \"{}\" in the HDF5.".format(dps_key))
```

Comment:
Maybe a check that the dps_key exists in the tractogram? Else, error, or else, warning?